### DISCUSSION TITLE
How to get verified on GitHub?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31285] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! Can someone help me? How to get verified on GitHub?

#### like this:
![image](https://user-images.githubusercontent.com/111701513/187317873-ee83ba8f-ba34-43fa-83a1-6d9bac8fd3ee.png)

What are the technical requirements required by GitHub for this?

**Comment by ghost** [comment, marked as answer]
Someone? This is possible?

**Reply by Zyztem** [reply]
This article explains what this means, in simple terms, it means they verified their domain; https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization
